tests/Model/PageTest.php

```diff
@@ -163,12 +163,8 @@ public function testGetterSetter(): void
         static::assertSame($time, $page->getCreatedAt());
         static::assertSame($time, $page->getUpdatedAt());
 
-        $children = [
-            new Page(),
-            new Page(),
-        ];
-
-        $page->setChildren($children);
+        $page->addChild(new Page());
+        $page->addChild(new Page());
```

Comment:
TBH I don't know why I cant do this:

```
        $children = new ArrayCollection([
            new Page(),
            new Page(),
        ]);

        $page->setChildren($children);
```

PHPStan complains:

```
  172    Parameter #1 $children of method Sonata\PageBundle\Model\Page::setChildren() expects
         Doctrine\Common\Collections\Collection<(int|string), Sonata\PageBundle\Model\PageInterface>,
         Doctrine\Common\Collections\ArrayCollection<int, Sonata\PageBundle\Tests\Model\Page> given.
```

Reply:
Weird since https://phpstan.org/r/ed0dcdc6-26c1-416c-8e99-012cfabd25f2